mep/common/management/export.py

```diff
@@ -146,7 +148,9 @@ def get_data(self, maximum=None):
         # grab the first N if maximum is specified
         if maximum:
             objects = objects[:maximum]
-        total = objects.count()
+        total = len(
+            objects
+        )  # fewer assumptions, allows other (multi model/class) objects
```

Comment:
was this needed? are we exporting anything other than database content in the new export scripts?

Reply:
ah, I see it's due to the address person/account issue; I'd like to resolve it there instead